src/Adapter/PlatformServices.Shared/netstandard1.3/Services/ns13TestContextPropertyStrings.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices
+{
+#pragma warning disable SA1649 // File name must match first type name
+
+    /// <summary>
+    /// Test Context Property Names.
+    /// </summary>
+    internal static class TestContextPropertyStrings
```

Comment:
How is this file different from : PlatformServices.Shared/netstandard1.0/Services/ns10TestContextPropertyStrings.cs
If both are same, consider removing this file.

Reply:
Okay. We can discuss if we want to keep this file or put it in contants.